Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -2750,51 +2896,40 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "116":
-    id: "116"
-    taskid: 98112215-f43b-4f78-875b-58f8da657f94
-    type: condition
+  "128":
+    id: "128"
+    taskid: fd248ad8-bee5-428e-8918-042e8b3865c8
+    type: regular
     task:
-      id: 98112215-f43b-4f78-875b-58f8da657f94
+      id: fd248ad8-bee5-428e-8918-042e8b3865c8
       version: -1
-      name: Are there service owners with emails?
-      description: Verifies if we have emails for service owners.
-      type: condition
+      name: Set playbook stage grid field (decision)
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of command:
```

Comment:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Instead of a value, you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```